test_package/CMakeLists.txt

```diff
@@ -1,12 +1,22 @@
 cmake_minimum_required(VERSION 3.1.0)
 project(test_package)
 
-include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
-conan_set_vs_runtime()
-conan_set_libcxx()
-conan_output_dirs_setup()
```

Comment:
Please keep these lines, this should solve the test recipe which currently fails

Reply:
Hmm...
My understanding of agreement done in https://github.com/bincrafters/conan-qt/pull/59 is that I should prove that fix in "conanfile.py" (not "test_package/conanfile.py" ) works for "cmake_find_package_multi" not "cmake".
Keeping those lines force/enable "cmake" (not "cmake_find_package_multi"), so we loose our goal here.

Also I can't see relation between those lines and unavailability `fontconfig` and `harfbuzz` packages.

Reply:
the fontconfig and harfbuzz problem originated from conan center index, and is now fixed.
Please takea look at the current errors:
https://github.com/bincrafters/conan-qt/pull/61/checks?check_run_id=1215031797
https://dev.azure.com/bincrafters/packages/_build/results?buildId=8363&view=logs&j=bb7a8683-d200-54be-ad2b-9054759b8844&t=77099861-29e4-59a5-7a01-0aed056ef6e0
https://dev.azure.com/bincrafters/packages/_build/results?buildId=8363&view=logs&j=6ac06546-f8ab-58ab-23a8-8b2d71391f05&t=9dfa956b-c14c-5be8-5a1e-9110788c36fd

If you really cannot use both `cmake` and `cmake_find_package_multi` generators, then someone needs to find another fix to these test_package errors.

Reply:
Done